modin/pandas/test/test_io.py

```diff
@@ -406,11 +397,29 @@ def test_read_csv_skipinitialspace(self):
 
         eval_io_from_str(str_initial_spaces, unique_filename, skipinitialspace=True)
 
-    @pytest.mark.xfail(reason="infinite recursion error - issue #2032")
     @pytest.mark.parametrize(
-        "test_case", ["single_element", "single_column", "multiple_columns"]
+        "test_case",
+        [
+            pytest.param(
+                "single_element",
+                marks=pytest.mark.xfail(
+                    reason="infinite recursion error - issue #2032"
+                ),
+            ),
+            pytest.param(
+                "single_column",
+                marks=pytest.mark.xfail(
+                    reason="infinite recursion error - issue #2032"
+                ),
+            ),
+            "multiple_columns",
+        ],
     )
-    def test_read_csv_squeeze(self, test_case):
+    def test_read_csv_squeeze(self, request, test_case):
```

Comment:
Why does the method signature change here?

Reply:
It fails cloud, and we detect cloud via `request`.